ImageItem/tests/ImageItem-specs.js

```diff
@@ -48,6 +48,16 @@ describe('ImageItem', () => {
 		expect(actual).toHaveTextContent(expected);
 	});
 
+	test('should support `children` prop when \'data-index\' is set', () => {
```

Comment:
Good.
But, please add one more TC with data-index & label.
  ImageItem.js     |     100 |    96.55 |     100 |     100 | 258      
It will hit L258 of ImageItem.js